src/components/Counter.tsx

```diff
@@ -0,0 +1,45 @@
+import React, { useCallback } from 'react';
```

Comment:
To avoid repetition and certain discrepancies, you can use the counter component I created for the product page.

You can it from it here, #56 , and if it is merged, it would be simpler to integrate.

Reply:
I agree, if that's the case let me change the label then.